examples/advanced/component-cti/README.md

```diff
@@ -0,0 +1,40 @@
+## Component CTI Structure
+
+This example will show you one way to define component tokens in an easy way while still using the CTI attributes.
```

Comment:
Is this a sufficient explanation for the user problem that this solves? I think we should lay out the issue in clarity. A user may not understand the solution, but they should be able to identify with the problem to know that this is the example they should be looking at.

Reply:
I moved the 'why do this section' up here because it better explains the problem. I also added an example of what the problem is with a token file in here. Hopefully that explains the issue a bit better.